.gitattributes

```diff
@@ -67,3 +49,33 @@ Programs/test_frozenmain.h  linguist-generated=true
 *.html  diff=html
 *.py    diff=python
 *.md    diff=markdown
+
+# Generated files
+# https://github.com/github/linguist/blob/master/docs/overrides.md
+#
```

Comment:
Would you mind to add a quick documentation how to ignore generated files locally in git diff commands?

Reply:
That's the next two lines ;)

Reply:
On Discord, @erlend-aasland showed something like ``git diff :exclude:generated`, but I don't recall the command.

I understand that `git config diff.generated.binary true` does the same thing, but without having to specify it on the command line each time.

Reply:
The command I showed used a glob pattern instead of querying attributes:

```sh
$ git diff main -- . ":(exclude)*clinic*"  # Explicitly exclude files that matches the pattern "*clinic*"
$ git config diff.generated.binary true    # Configure git diff to treat files with the 'generated' attribute as binary files
$ git diff main                            # With the git config from above, git diff will treat all generated files as binary files
```

Reply:
With this change, there is now also the option to do something like `git diff -- ":(attr:!generated)"` to completely elide generated files from diff output.  I'm not sure if it's worth it to document those options here, though.

Reply:
Is there a git cheat sheet in the devguide? We could add some tips and tricks there.

Reply:
I created https://github.com/python/devguide/issues/794